docs/setup/connect-to-elasticsearch.asciidoc

```diff
@@ -1,108 +1,75 @@
+add a sample data set and give {kib} a test drive.
+The built-in options are available on the home page.
 
-* Upload a CSV, JSON, or log file with the File Data Visualizer.
+[role="screenshot"]
+image::images/add-data-home.png[Built-in options for adding data to Kibana:  Add data, Add Elastic Agent, Upload a file]
 
-* Upload geospatial data with the GeoJSON Upload feature.
+[float]
+[[add-data-tutorial-kibana]]
+===  Add data
 
-* Index logs, metrics, events, or application data by setting up a Beats module.
+Want to ingest logs, metrics, security, or application data?
+Install and configure a Beats data shipper or other module to periodically collect the data
+and send it to {es}. You can then use the pre-built dashboards to explore and analyze the data.
 
-* Connect {kib} with existing {es} indices.
+[role="screenshot"]
+image::images/add-data-tutorials.png[Add Data tutorials]
 
-If you're not ready to use your own data, you can add a <<get-started, sample data set>>
-to see all that you can do in {kib}.
+[discrete]
+=== Add Elastic Agent
 
-[float]
-[[upload-data-kibana]]
-=== Upload a CSV, JSON, or log file
+beta[] *Elastic Agent* is a centralized way to set up your data integrations.
+With the *Fleet* UI, you can add
+and manage integrations for popular services and platforms, providing
+an easy way to collect your data. The integrations
+ship with dashboards and visualizations,
+so you quickly get insights on your data.
 
-experimental[]
+To get started, refer to
+{ingest-guide}/ingest-management-getting-started.html[Quick start: Get logs and metrics into the Elastic Stack].
 
-To visualize data in a CSV, JSON, or log file, you can upload it using the File 
-Data Visualizer. On the home page, click *Upload a file*, and 
-then drag your file onto the *File Data Visualizer*. Alternatively, you can open 
-it by navigating to *Machine Learning* from the side navigation and selecting 
+[role="screenshot"]
+image::images/add-data-fleet.png[Add data using Fleet]
 
-*Data Visualizer*.
+NOTE: This feature is not intended for use as part of a
+repeated production process, but rather for the initial exploration of your data.
 
-[role="screenshot"]
-image::images/ingest-data.png[File Data Visualizer on the home page]
+[discrete]
+[[upload-data-kibana]]
+=== Upload a file
 
-You can upload a file up to 100 MB. This value is configurable up to 1 GB in
+experimental[] If your data is in a CSV, JSON, or log file, you can upload it using the File
+Data Visualizer. You can upload a file up to 100 MB. This value is configurable up to 1 GB in
 <<kibana-ml-settings, Advanced Settings>>.
 
 [role="screenshot"]
 image::images/add-data-fv.png[File Data Visualizer]
 
-The File Data Visualizer uses the {ref}/ml-find-file-structure.html[find_file_structure API] to analyze
-the uploaded file and to suggest ingest pipelines and mappings for your data.
+For geospatial data, you can use the File Data Visualizer or the
```

Comment:
I don't think you need to call out File Data Visualizer or GeoJSon Uploader. How about just changing to "See <<import-geospatial-data, Import geospatial data>> for uploading files with geospatial data."

Reply:
++